oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/endpoint/AuthorizationCodeTokenRequestAttributes.java

```diff
@@ -28,6 +29,7 @@
 	private String code;
 	private String clientId;
 	private String redirectUri;
+	private AuthorizationGrantType grantType;
```

Comment:
The grant type member is not needed here as it's implied based on the class name -> `AuthorizationCodeTokenRequestAttributes`. Can you remove this please.